tool/healthz/main.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2017 Gravitational, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/gravitational/trace"
+	minio "github.com/minio/minio-go"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	defaultEndpoint       = "localhost:18080"
+	defaultBucket         = "liveness-check"
+	defaultBucketLocation = "G1"
+	defaultPrefix         = "liveness"
+)
+
+// Config for access to S3 storage
+type s3Config struct {
+	bucket string
+	client *minio.Client
+}
+
+func main() {
+	s3AccessKeyID := flag.String("access-key", "", "S3 access key")
```

Comment:
For compatibility with `AWS`, I'd call these `access key id` and `secret access key`